src/context/wallet/context.tsx

```diff
@@ -1,10 +1,11 @@
 import {createContext, useContext} from "react";
+import {WalletNetworks} from "../../constants";
 
 export interface walletContext {
   isInstalled: boolean;
   isConnected: boolean;
   isAccountSet: boolean;
-  walletNetwork: string;
+  walletNetwork: WalletNetworks;
```

Comment:
Does it make sense to limit the scope of the wallet-specific network names (i.e. change this to `NetworkName` (the explorer network name, and getWalletNetwork() could also return NetworkName))?

Reply:
I thought about it but it might be a bit confusing to set the `walletNetwork` (i.e Localhost/Devnet/Testnet) as an `explorerNetwork` (i.e local/devnet/test) where in fact they are different, it might makes more sense to have it as it really is (i.e Localhost/Devnet/Testnet), and when we need (for now for the validation) we can use the map we just created